port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -68,15 +74,19 @@ async def _search_as_object(
 
         return result
 
-    async def _get_entity_if_passed_selector(
+    async def _get_mapped_entity(
         self,
         data: dict[str, Any],
         raw_entity_mappings: dict[str, Any],
         selector_query: str,
+        ignore_selector: bool = False,
```

Comment:
there is inconsistency with the parameter across all the functions that pass it through.